aioftp/server.py

```diff
@@ -857,6 +859,7 @@ def __init__(self,
             loop=self.loop
         )
         response_queue = asyncio.Queue(loop=self.loop)
+        get_current_task = asyncio.current_task if IS_PY37 else asyncio.Task.current_task
```

Comment:
`get_current_task` also can be moved on top level. And I vote for less foggy "if else". No need to make oneliner.

Reply:
Done!